packages/pluggableWidgets/document-viewer-web/src/DocumentViewer.tsx

```diff
@@ -0,0 +1,28 @@
+import { ReactNode, createElement } from "react";
+import { DocumentViewer as DocumentViewerComponent } from "./components/DocumentViewer";
+import { DocumentViewerContainerProps } from "../typings/DocumentViewerProps";
+import { useFileURL } from "./utils/useFileURL";
+import { useStyle } from "./utils/useStyle";
+import "./ui/DocumentViewer.scss";
+
+export function DocumentViewer(props: DocumentViewerContainerProps): ReactNode {
+    const result = useFileURL(props);
+    const style = useStyle(props);
+
+    if (result.loading) {
+        return null;
```

Comment:
it's better to show some progress icon in this case, u can check examples in `richtext` and `datagrid`. it's one icon and css styles